vue-app/src/components/RoundInformation.vue

```diff
@@ -2,7 +2,19 @@
   <div class="projects">
     <div class="round-info">
       <div class="image-wrapper">
-        <img src="@/assets/docking.png" height="100%" />
+        <img
+          src="@/assets/docking/docking.png"
```

Comment:
We would need the original images to be optimized too in case the browser do not support these attrs.

Reply:
I agree, this is great but maybe a more optimized image for the default. Ill make some code suggestions, but other than that this is great!

Reply:
```suggestion
          src="../assets/docking/docking_w1080.png"
```